### ISSUE TITLE
Can't copy and paste the result out of the browser

### ISSUE THREAD

**Issue by geekpete** [astashov/tixi#21]
Tested on:
- Firefox 72.0.2 on Linux
- Chromium Version 79.0.3945.130 (openSUSE Build) (64-bit)
 
Paste not working and no other way to export means the tool can't be used.
A simple Save-as or Export functionality would be great.

**Comment by astashov** [comment]
Does this icon work?

![image](https://user-images.githubusercontent.com/12795/73376356-ef86ec00-4282-11ea-958a-558aae2a7a96.png)

It should give you copiable output.

**Comment by geekpete** [comment]
Ah :man_facepalming: I just thought this was like a preview/render button that wasn't working or something, never saw the copy/paste text below, I was trying to select in the main interface and copy from there which wasn't working.

I might submit a tiny PR to add a one-page super basic usage guide with screenshots if you're cool with that? To make it totally obvious how everything works.

This tool is really handy for sketching quick [Ditaa](http://ditaa.sourceforge.net/) diagrams which I then put through [asciidoctor-diagram](https://asciidoctor.org/docs/asciidoctor-diagram/) to create nice diagrams that I can store/version with git.

**Comment by geekpete** [comment]
It might be cool if that icon was a more recognizable "Save" button that popped up the text in the same window but also had an Export/Save-As button to download the result.

**Comment by astashov** [comment]
Yeah, sure, though the version deployed is currently differs from what is in this repo - it's written in Elm now. I need to update the repo it sometime to make it up to date...

**Comment by GuybrushX** [comment]
I can confirm that CTRL + C doesn't work (anymore?) after clicking that eye icon.

Running Chrome on Windows 10. I had to select the diagram first with the mouse like normal text. Maybe that's intended but not very intuitive? I expected to just press "CTRL + C in the pop up without selecting text first.

**Comment by lawrenceong001** [comment]
hi @astashov -- do you know when you can update the repo? I'd like to host this myself. 

thanks.

**Comment by maelvls** [comment]
> It's written in Elm now. I need to update the repo it sometime to make it up to date...

Hi @astashov, thank you so much for your work on this!! Do we have an ETA for the update of the repo? I wish to contribute, and I guess it would make sense to have the latest changes to do so 😇
